docs/book/src/tasks/installation.md

```diff
@@ -176,55 +33,32 @@ export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
 export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"
 ```
 
-```bash
-curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-azure" asset:"infrastructure-components.yaml" version:"0.3.x"}} \
-  | envsubst \
-  | kubectl create -f -
-```
+For more information about authorization, AAD, or requirements for Azure, visit the [Azure Provider Prerequisites](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/getting-started.md#prerequisites) document.
 
 {{#/tab }}
 {{#tab Docker}}
 
-Check the [Docker provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-docker/releases) for an up-to-date components file.
-
-```bash
-kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-docker" asset:"provider-components.yaml" version:"0.2.x"}}
-```
+If you are planning to use to test locally Cluster API using the Docker infrastructure provider, please follow additional
+steps described in the [developer instruction](../clusterctl/developers.md#additional-steps-in-order-to-use-the-docker-provider) page.
 
 {{#/tab }}
 {{#tab GCP}}
 
-<aside class="note warning">
-
-<h1>Action Required</h1>
-
-Update the path to your GCP credentials file below.
-
-</aside>
-
-##### Create the components
-
-Check the [GCP provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/releases) for an up-to-date components file.
-
 ```bash
 # Create the base64 encoded credentials by catting your credentials json.
 # This command uses your environment variables and encodes
 # them in a value to be stored in a Kubernetes Secret.
 export GCP_B64ENCODED_CREDENTIALS=$( cat /path/to/gcp-credentials.json | base64 | tr -d '\n' )
-
-# Create the components.
-curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-gcp" asset:"infrastructure-components.yaml" version:"0.3.x"}} \
-  | envsubst \
-  | kubectl create -f -
 ```
 
 {{#/tab }}
 {{#tab vSphere}}
 
 It is required to use an official CAPV machine image for your vSphere VM templates. See [Uploading CAPV Machine Images](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md#uploading-the-capv-machine-image) for instructions on how to do this.
 
+Then, it is required Upload vCenter credentials as a Kubernetes secret:
```

Comment:
This assumes that a Kubernetes cluster is already available, but those steps aren't explained until later in this doc now.

Reply:
This still is still trying to apply a k8s secret to a cluster that does not yet exist (or is not yet configured), it might be good to move the kind/kubeconfig steps prior to the provider components so that this has a cluster to apply to.

Reply:
done!